syft/src/main/java/org/openmined/syft/datasource/JobLocalDataSource.kt

```diff
@@ -114,4 +114,14 @@ internal class JobLocalDataSource {
         }
         return file.absolutePath
     }
+
+    @ExperimentalUnsignedTypes
+    fun getModelsPath(config: SyftConfiguration) = "${config.filesDir}/${dataModel.id}/models"
```

Comment:
I think it is better to return the complete path here. Give model id as an argument and return the complete path